frameworks/PHP/hamlet/hamlet.dockerfile

```diff
@@ -1,22 +1,22 @@
-FROM ubuntu:18.10
+FROM debian:stretch
 
```

Comment:
Any reason to use debian:stretch ??

I's easier use the same when we make changes and also all use the same stack in the bench.

Reply:
@joanhey revereted back to ubuntu

Reply:
Thank you.
Now it's better for the docker cache.
Also debian stretch have nginx 1.10.3.

And the next week we will change to ubuntu 19.04, that come with nginx 1.15.9.
From 1.15.6 have fastcgi_socket_keepalive, and we will try it.